07-Types-of-contributions.md

```diff
@@ -0,0 +1,173 @@
+
+Example: [Improved the user experience of the highlights input form](https://github.com/open-sauced/app/pull/1362)
+
+- **Accessibility Design**: Ensuring that the project's graphical elements are accessible to all users, including those with disabilities, is an important aspect of graphic design. Designers can create accessible color schemes and ensure that visual elements are perceivable and operable.
+
+Example: [Redesign footer
+](https://github.com/cesko-digital/pomahejukrajine-web/pull/222)
+
+- **Responsive Design**: Designing visuals that work well on various devices and screen sizes is crucial for modern web applications. Responsive design ensures that the project looks good and functions properly on both desktop and mobile devices
+
+Example: [Writing about an OSS project creates more visibility for the project and the maintainer, which can lead to better discoverability and support.](https://github.com/open-sauced/app/pull/1553)
+
+### Blog Writing
+
+If you're more of a creative writer, then blog writing is your perfect match. This type of open-source contribution involves creating and publishing blog posts, articles, or tutorials related to open-source software, projects, technologies, or best practices. Like documentation, you don't necessarily have to know how to code to write these posts.
+
+### Ways to use Blogging as a Open Source Contribution
+
+- You can write about the latest features of an open-source project, such as a new version of the software or a new plugin.
+
+Example: [Announcing the 0.2.0 release of Analog](https://dev.to/analogjs/announcing-the-020-release-of-analog-aa1)
+
+- Love teaching? You can write a tutorial on how to install and configure the software.
+
+Example: [How To Make a Delicious Contributing Guide](https://dev.to/opensauced/how-to-make-a-delicious-contributing-guide-4bp3)
+
+- A blog post could be written about the impact of an open-source project, such as how it has been used to make a difference in the world.
+
+Example: [Retaining the best engineers](https://opensauced.pizza/blog/retaining-the-best-engineers)
+
+- A blog post could be written on how to navigate your way around open-source as a new contributor, such as how to get started with open-source.
+
+Example: [Open Source 101: A Beginner's Guide to Getting Started](https://opensauced.pizza/blog/open-source-101-a-beginner's-guide-to-getting-started)
+
+## Coding Type Of Contributions
+
+Coding contributions are the heart of open-source development. They entail writing, modifying, or improving the actual source code of an open-source project, directly impacting the functionality and quality of the software. Contributors collaborate with project maintainers and the community to ensure that their code contributions align with project goals and coding standards. Also, open-source projects typically have guidelines and processes for accepting and integrating coding contributions, such as code reviews and continuous integration.
```

Comment:
I have a hard time thinking of it as the heart. And tbh, I may have said this before, but I think we should adjust our language here.

Coding contributions power open-source development.